src/main/go/org/jenkinsci/plugins/durabletask/generate-binaries.sh

```diff
@@ -12,6 +12,11 @@ docker build --build-arg PLUGIN_VER=${VER} -t ${IMG_NAME}:${VER} .
 docker run -i --rm \
     --mount type=bind,src=${SRC},dst=/org/jenkinsci/plugins/durabletask \
     ${IMG_NAME}:${VER}
+docker rmi ${IMG_NAME}:${VER}
 mkdir -p ${DST}
 mv ${BIN_NAME}_* ${DST}/
-docker rmi ${IMG_NAME}:${VER}
\ No newline at end of file
+if [ $? -ne 0 ]
```

Comment:
Like we discussed, I think `set -e` might be better than manually checking errors.

Reply:
addressed in 1e06b6e